packages/shared/lib/services/proxy.service.ts

```diff
@@ -36,7 +37,8 @@ class ProxyService {
             }
         }
 
-        const { endpoint: passedEndpoint, providerConfigKey, connectionId, method, retries, data, headers, baseUrlOverride } = externalConfig;
+        let data = externalConfig.data;
+        const { endpoint: passedEndpoint, providerConfigKey, connectionId, method, retries, headers, baseUrlOverride } = externalConfig;
```

Comment:
why moving data out of the attributes matching?

Reply:
Because it needs to be a `let` to allow for possible reassignment